seesaw/seesaw.go

```diff
@@ -0,0 +1,113 @@
+package seesaw
+
+import (
+	"errors"
+	"strconv"
+	"time"
+)
+
+const DefaultSeesawAddress = 0x49
+
+// empirically determined delay, the one from the official library seems to be too short (250us)
+const defaultDelay = 10 * time.Millisecond
```

Comment:
I'd be happy to have a better alternative. That's also how the official library does it :/